src/ua/griddynamics/Main.java

```diff
@@ -0,0 +1,40 @@
+package ua.griddynamics;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("How many mines do you want on the field? >");
+        int numberOfMines = scanner.nextInt();
+
+        Game game = new Game(numberOfMines);
+        System.out.println(game.getField());
+        CoordinateChecker coordinateChecker = new CoordinateChecker();
+
+        boolean isMinesPlaced = false;
+        while (!game.isWin(numberOfMines)) {
+            String[] userAnswer = coordinateChecker.scanAndCheckUserInput();
+            Coordinate coordinate = coordinateChecker.getCoordinate(userAnswer, game);
+            if (!isMinesPlaced) {
+                game.placeMines(coordinate);
+            }
+            isMinesPlaced = true;
+            game.putAnswer(coordinate);
```

Comment:
Why do we call `game.putAnswer(coordinate)` 3 times? 
lines 23, 24, 30

Reply:
Done